scripts/run-orderly-web-deps.sh

```diff
@@ -10,12 +10,13 @@ root=$(realpath $here/..)
 # create orderly db
 if [[ -d $root/src/demo ]]
 then
-  echo "Orderly demo folder already exists, not re-creating it."
```

Comment:
The build was also failing here but I don't really understand why - first it was complaining about missing the demo db artifact from the orderly build because all the TC data had been lost, so I just removed the dependency on that build (seeing as we're migrating away from TC anyway), assuming that it would hit this condition and regenerate the demo db. But it was hitting this condition and finding that `/src/demo` already existed and I still can't for the life of me see why/how. I've changed it to regenerate the demo db every time but I still don't understand why this wasn't working as is!

Reply:
Is it hitting this code twice in the course of the TC builds? Once in blackbox tests and once from ..something else?

Reply:
That's what I wondered but couldn't see how it was